cre.sh

```diff
@@ -148,11 +151,15 @@ function f_make_report
     
     if [ "$type" == "denovo" ]
     then
-	export denovo=1
+			export denovo=1
     fi
 
-    cre.gemini2txt.vcf2db.sh ${family}-ensemble.db $depth_threshold $severity_filter $max_af > $family.variants.txt
-    cre.gemini.variant_impacts.vcf2db.sh ${family}-ensemble.db $depth_threshold $severity_filter $max_af > $family.variant_impacts.txt
+    cre.gemini2txt.vcf2db.sh ${family}-ensemble.db $depth_threshold $severity_filter $max_af > $family.variants.all.txt
+    cre.gemini.variant_impacts.vcf2db.sh ${family}-ensemble.db $depth_threshold $severity_filter $max_af > $family.variant_impacts.all.txt
+
+    # remove duplicate lines from results of gemini query
```

Comment:
Why are we seeing duplicate lines from the gemini query? Should a unique clause be added to the query itself?

Reply:
two queries, one for clinvar and one for regular so even if unique is put on both of them, duplicates will be present